pkgs/development/tools/misc/indent/default.nix

```
@@ -10,14 +10,17 @@ stdenv.mkDerivation rec {
   };
 
   patches = [ ./darwin.patch ];
+  makeFlags = [ "AR=${stdenv.cc.targetPrefix}ar" ];
 
-  buildInputs = [ texinfo ];
+  nativeBuildInputs = [ texinfo ];
```

Now that this works with static compilation (and other types of cross-compilation as well) might be a good idea to set strictDeps to mitigate regressions

```suggestion
  strictDeps = true;
  nativeBuildInputs = [ texinfo ];
```